waffle/admin.py

```diff
@@ -18,22 +18,22 @@ def enable_for_all(ma, request, qs):
     for f in qs.all():
         f.everyone = True
         f.save()
-enable_for_all.short_description = 'Enable selected flags for everyone.'
+enable_for_all.short_description = 'Enable selected flags for everyone.'  # noqa: E305,E501
```

Comment:
Is there a way to fix these without opting out of the linter? E305 is missing blank lines and E501 is line too long, right?